### ISSUE TITLE
[FEAT] Cloud backup through `git`

### ISSUE THREAD

**Issue by heckarman** [quillpad/quillpad#315]
Hey there!

Have been using Quillpad for well I over an year now. I use Obsidian on my PC and Mac. Obsidian Android app for some reason is not intuitive and the git plugin does not work.

My proposal here is, an option to set up Git for backup of notes. The params would either be auth and signing keys or PAT, GitHub.

I would want to take this up and work on this. The concern that I've is file format of QuillPad is completely different from that of Obsidian which would require a lot of changes to be done. If done well, this would fix performance issues like what #309 states.

**Comment by githubcatw** [comment]
I think this will only be possible once milestone 1.5 is reached.

**Comment by shuvashish76** [comment]
Yes supporting sync with `git` providers e.g. GitHub, GitLab, Codeberg, Gitea, SourceHut etc would be nice.

For example see:
- https://f-droid.org/en/packages/org.stridetasks.stride/
- https://f-droid.org/en/packages/org.handmadeideas.floccus/

**Comment by rvyhvn** [comment]
any update?
